apps_download/views/product_template_view.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright 2013-2017 BizzAppDev - Ruchir Shukla <ruchir@bizzappdev.com>
+    Copyright 2013-2017 AgentERP - Georg Notter <georg.notter@agenterp.com>
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <data>
```

Comment:
Can remove data tag, this is implicit with the `odoo` tag, and is only required if mixing noupdate=1 and noupdate=0 datasets in the same file